src/main.rs

```diff
@@ -375,11 +382,10 @@ fn main() {
 			.filter_map(|(name, loc)| match loc {
 				gb_sym_file::Location::Banked(bank, addr) => Some((name, (bank, addr))),
 				_ => None,
-			})
-			.collect()
-	} else {
-		HashMap::new()
-	};
+			});
+		symfile.extend(symbols);
+	}
+	let symfile = symfile;
```

Comment:
Does this do anything?? If it *does*, this is very unclear and I feel like it should be written differently.

Reply:
No, it just "changes" the mutability. This would have the same effect:

```rs
let symfile = {
  let mut symfile = HashMap::new();
  // ...
  symfile
};
```

Reply:
it removes the `mut` capability :) basically "from now on, `symfile` is a const". if the previous comment on the declaration of `symfile` goes through then this would be eliminated.